src/features/profile/components/ProfileHeader.tsx

```diff
@@ -70,13 +71,34 @@ export function ProfileHeader(props: ProfileHeaderProps) {
       />
     )
   }
+  if (user.isEligibleForBeneficiaryUpgrade) {
+    return <EligibleForBeneficiaryUpgradeHeader />
+  }
+  if (today < user.eligibilityStartDatetime) {
+    return <YoungerBadge />
+  }
+
+  return <BodyContainer testID="body-container-above-18" padding={1} />
+}
+
+const EligibleForBeneficiaryHeader = () => {
+  const [nextSubscriptionStep, setNextSubscriptionStep] = useState(undefined)
+
+  useEffect(() => {
+    setNextSubscriptionStep(fetchNextSubscriptionStep())
+  }, [])
+
+  if (nextSubscriptionStep === undefined) {
+    return <Loader />
+  }
+
+  if (nextSubscriptionStep !== null) {
+    return <div onClick={navigateToNextStep}>Profite de 300€</div>
+  }
+
+  if (user.subscriptionMessage !== null) {
+    return <IdCheckProcessingBadge subscriptionMessage={user.subscriptionMessage} />
+  }
 
-  return (
-    <BeneficiaryHeader
-      firstName={user.firstName}
-      lastName={user.lastName}
-      domainsCredit={user.domainsCredit}
-      depositExpirationDate={displayedDepositExpirationDate}
-    />
-  )
+  return <div>Dossier en attente</div>
```

Comment:
Hmm mais ça c'est un message in app en qlq sorte non, on le récupère pas du back ? Je serai plus d'avis de ne rien afficher

Reply:
Oui c'est un affichage par défaut... Normalement on ne devrait pas arriver dans ce cas mais si c'est le cas c'est qu'il y a un dossier en attente